src/test/_impl/rounding/convert_operator.cpp

```diff
@@ -82,3 +90,47 @@ namespace test_convert_nearest_rounding_scaled_integer
     static_assert(identical(cnl::scaled_integer<int, cnl::power<-2>>{0.25}, c),
             "cnl::convert<nearest_rounding_tag, scaled_integer, scaled_integer>");
 }
+
+namespace  test_convert_towards_infinity_rounding_native_datatypes
+{
+
+    static_assert(
+            cnl::_impl::identical(0.123F, cnl::convert<cnl::towards_infinity_rounding_tag, float, float>(0.123F)),
+            "cnl::convert<towards_infinity_rounding_tag, float, float>");
+
+    static_assert(
+            cnl::_impl::identical(0.125, cnl::convert<cnl::towards_infinity_rounding_tag, double, float>(0.125F)),
+            "cnl::convert<towards_infinity_rounding_tag, double, float>");
+
+    static_assert(
+            cnl::_impl::identical(0, cnl::convert<cnl::towards_infinity_rounding_tag, int, float>(0.125F)),
+            "cnl::convert<towards_infinity_rounding_tag, int, float>");
+
+    static_assert(
+            cnl::_impl::identical(1, cnl::convert<cnl::towards_infinity_rounding_tag, int, float>(0.5F)),
+            "cnl::convert<towards_infinity_rounding_tag, int, float>");
+
+    static_assert(
+            cnl::_impl::identical(1, cnl::convert<cnl::towards_infinity_rounding_tag, int, float>(0.725F)),
+            "cnl::convert<towards_infinity_rounding_tag, int, float>");
+
```

Comment:
Suggested test for completeness:

```c++
    static_assert(
            cnl::_impl::identical(-1, cnl::convert<cnl::towards_infinity_rounding_tag, int, float>(-0.125F)),
            "cnl::convert<towards_infinity_rounding_tag, int, float>");
```

Reply:
This should round to 0
    static_assert(
            cnl::_impl::identical(0, cnl::convert<cnl::towards_infinity_rounding_tag, int, float>(-0.125F)),
            "cnl::convert<towards_infinity_rounding_tag, int, float>");